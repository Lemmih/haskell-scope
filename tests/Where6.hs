module Where6 where

a x = b
  where b = b

a' x = b
  where b = b
