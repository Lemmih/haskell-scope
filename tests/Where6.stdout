Scope errors:

Definitions:
  Definition 1, from Where6.decl0:
    [4ma x = b[0m
    [4m  where b = b[0m
  Definition 2, from Where6.decl0.match0:
    a [4mx[0m = b
  Definition 3, from Where6.decl0.match0.rhs.binds0:
      where [4mb[0m = b
  Definition 4, from Where6.decl1:
    [4ma' x = b[0m
    [4m  where b = b[0m
  Definition 5, from Where6.decl1.match0:
    a' [4mx[0m = b
  Definition 6, from Where6.decl1.match0.rhs.binds0:
      where [4mb[0m = b

Use sites:
  Definition used: 1
    [4ma[0m x = b
  Definition used: 3
    a x = [4mb[0m
  Definition used: 3
      where b = [4mb[0m
  Definition used: 4
    [4ma'[0m x = b
  Definition used: 6
    a' x = [4mb[0m
  Definition used: 6
      where b = [4mb[0m
